package.json

```diff
@@ -13,23 +13,23 @@
     "typescript": "^4"
   },
   "engines": {
-    "node": ">=16.0.0",
-    "npm": ">=7.0.0"
+    "node": ">=16.13.0",
+    "npm": ">=8.1.0"
   },
   "scripts": {
+    "actions": "find . -name '*_action.sh' | sed -E 's:./src/(.*)_action.sh:\\1:' | grep -v 'scripts/do_action'",
     "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts src/server_manager/install_scripts/gcp_install_script.ts third_party/shellcheck/download/",
     "do": "bash ./scripts/do_action.sh",
-    "actions": "find . -name '*_action.sh' | sed -E 's:./src/(.*)_action.sh:\\1:' | grep -v 'scripts/do_action'",
     "lint": "npm run shellcheck && npm run tslint",
-    "shellcheck": "bash ./scripts/shellcheck.sh",
+    "shellcheck": "bash ./scripts/check_shell.sh",
```

Comment:
The reason why it's called `shellcheck` is because it's the name of the tool. If anything, that's doing linting.

Perhaps we should rename it to `lint:sh`, and `tslint` to `lint:ts`. That's clearer and shows the relationship to `lint`.

Reply:
yes! I almost did `check:shell` but `lint:shell` is great too 🥰